Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -92,10 +92,10 @@
 {
     if (!m_webPageProxy.hasRunningProcess())
         return;
+    m_webPageProxy.contentsSizeChanged();
```

Comment:
`sizeDidChange()` is not the same as `contentsSizeChanged()`. The former means that the view was resized. The latter means that the scrolled contents got bigger or smaller.

Reply:
Changed these to viewSizeDidChange to make it more clear. contentsSizeChanged might need to be plumbed through the proxy scrollbar object from the web process.